<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineParamForm</class>
 <widget class="QWidget" name="LineParamForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditX1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelX1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelX2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x2 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditX2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelY1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y1 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditY1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelY2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditY2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
